app/screens/contributer_screen.rb

```diff
@@ -11,6 +11,7 @@ def on_load
   end
 
   def on_refresh
+    stop_refreshing
```

Comment:
yeah i started reviewing ProMotion a bit to see what was going on here - I am definitely more aware of the RMQ space at this point.